### PR TITLE
Add test for global pointer

### PR DESCRIPTION
**Pull request by Smit-create**

### CODE REVIEW

File: integration_tests/CMakeLists.txt [hunk 1]
```diff
@@ -115,6 +115,7 @@ endmacro(RUN)
 # Test zero and non-zero exit code and assert statements
 RUN(NAME bindc_01            LABELS llvm c)
 RUN(NAME bindc_02            LABELS llvm c)
+RUN(NAME bindc_04            LABELS llvm)
```
Comment (czgdp1807) [inline integration_tests/CMakeLists.txt:118]: Let's remove it from here as `bindc_04` won't work with LLVM backend right now.

File: integration_tests/bindc_04.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+x: Pointer[i16[:]]
+
+def f():
+    yptr1: Pointer[i16[:]]
+    y: i16[2]
+    y[0] = 1
+    y[1] = 2
+    yptr1 = pointer(y)
+    assert yptr1[0] == 1
+    assert yptr1[1] == 2
```
Comment (czgdp1807) [inline integration_tests/bindc_04.py:13]: Using `ArrayRef` directly in `assert` doesn't work yet. I will fix this next week.

File: integration_tests/bindc_04.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    x = pointer(y)
+
+def check():
+    f()
+    assert x[0] == 1
+    assert x[1] == 2
```
Comment (czgdp1807) [inline integration_tests/bindc_04.py:19]: +1.

P.S. No changes needed in the test.

**Comment by czgdp1807** [comment]
`bindc_04` works with LLVM. Surprising.

File: integration_tests/CMakeLists.txt (hunk 1 above)
Comment (czgdp1807) [reply integration_tests/CMakeLists.txt:118]: It works. Ignore.

**Review by czgdp1807 — APPROVED** [review]

**Comment by czgdp1807** [comment]
@certik This looks good to me. Any thoughts from your end.

**Review by certik — APPROVED** [review]
I think this is fine.

**Pull request merged**